node/src/chain_spec.rs

```diff
@@ -179,11 +186,27 @@ fn testnet_genesis(
 		},
 		// no need to pass anything to aura, in fact it will panic if we do. Session will take care
 		// of this.
-		pallet_aura: manta_pc_runtime::AuraConfig {
-			authorities: invulnerables.iter().map(|v| v.1.clone()).collect(),
-		},
+		pallet_aura: Default::default(),
 		pallet_sudo: manta_pc_runtime::SudoConfig { key: root_key },
 		parachain_info: manta_pc_runtime::ParachainInfoConfig { parachain_id: id },
+		pallet_collator_selection: manta_pc_runtime::CollatorSelectionConfig {
+			invulnerables: invulnerables.iter().cloned().map(|(acc, _)| acc).collect(),
+			candidacy_bond: MA * 16,
+			..Default::default()
+		},
+		pallet_session: manta_pc_runtime::SessionConfig {
+			keys: invulnerables
+				.iter()
+				.cloned()
+				.map(|(acc, aura)| {
+					(
+						acc.clone(),                 // account id
+						acc.clone(),                 // validator id
```

Comment:
Guess validator and account are the same thing here, right ?

Reply:
Yes, but we can configure it in the future.